src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -855,6 +856,36 @@ private void Logic_NodeMessagesClearing(NodeModel obj)
             }
         }
 
+        /// <summary>
+        /// Clears the existing messages on a node before it executes and re-evalutes its warnings/errors. 
+        /// </summary>
+        /// <param name="obj"></param>
+        private void Logic_NodeInfoMessagesClearing(NodeModel obj)
+        {
+            if (ErrorBubble == null) return;
+
+            if (DynamoViewModel.UIDispatcher != null)
+            {
+                DynamoViewModel.UIDispatcher.Invoke(() =>
+                {
+                    var itemsToRemove = ErrorBubble.NodeMessages.Where(x => x.Style == InfoBubbleViewModel.Style.Info).ToList();
```

Comment:
Is the ToList required here and below?

Reply:
Addressed this in the new PR: https://github.com/DynamoDS/Dynamo/pull/12734